src/thing/get.ts

```diff
@@ -281,6 +281,43 @@ export function getStringWithLocaleAll(
   return matchingQuads.map((quad) => quad.object.value);
 }
 
+/**
+ * Retrieves all string literals for the specified property from the specified
+ * Thing.
+ *
+ * NOTE: Assumes you also want non-locale string literals too (i.e. literals
+ * with the xsd:string datatype). These values will be returned in a map entry
+ * with a key of the empty string.
+ *
+ * @param thing The [[Thing]] to read the localised string values from.
+ * @param property The given Property for which you want the localised string values.
+ * @returns A Map of objects, keyed on locale with the value an array of string values (for that locale).
+ */
+export function getStringByLocaleAll(
+  thing: Thing,
+  property: Url | UrlString
+): Map<string, string[]> {
```

Comment:
Could you add to the README that people might need a polyfill for `Map` as well?

Reply:
`Map` feels like the correct structure here (as opposed to `Array`), but no super-strong opinion either way. It's just when I ask for something 'by' some key, I kinda expect to be able to then lookup by a key value easily - i.e. using `Map.get(key)` instead of having to iterate through an array.